src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -1621,5 +1622,55 @@ class C
                 End Try
             End Using
         End Function
+
+        <WorkItem(123188)>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TestMatchWithTurkishIWorkaround1() As Task
+            Dim originalCulture = CultureInfo.CurrentCulture
+            Dim trCulture = New CultureInfo("tr-TR")
+            System.Threading.Thread.CurrentThread.CurrentCulture = trCulture
```

Comment:
Isn't there already infrastructure available for doing this in the compiler unit tests? If not, we should consider factoring this into a helper rather than copying and pasting the same Try..Catch into each test.